### ISSUE TITLE
IDE0009 never reported for methods in VB

### ISSUE THREAD

**Issue by sharwell** [dotnet/roslyn#19750]
**Version Used**: 15.2, 15.3 Preview 1

**Steps to Reproduce**:

1. Set the VB option for **Qualify method access with 'Me'** to **Prefer 'Me.'**

    ![image](https://cloud.githubusercontent.com/assets/1408396/26413707/702f0e2a-4072-11e7-8c3b-abaa199f135a.png)

2. Add the following code

    ```vb
    Class SomeType
        Overridable Function Test1() As Integer
            Return 0
        End Function

        Overridable Function Test2() As Integer
            Return 0
        End Function

        Overridable Sub Caller()
            Test1()
            Test2()
        End Sub
    End Class
    ```

**Expected Behavior**:

IDE0009 reported on the call sites for `Test1` and `Test2`

**Actual Behavior**:

IDE0009 not reported at all

**Comment by Pilchie** [comment]
Putting in `15.later`, but if a fix from the community appeared "soon", we might be able to squeeze it into 15.3.

**Comment by Pilchie** [comment]
Closing as a dupe of #19373.
